tokens/properties/components/Dropdown.json

```diff
@@ -0,0 +1,221 @@
+            "font-family": {
+                "value": "{font.family.style.label.value}"
+            },
+            "font-size": {
+                "value": "{font.size.style.label.value}"
+            },
+            "font-weight": {
+                "value": "{font.weight.bold.value}"
+            },
+            "line-height": {
+                "value": "{font.line-height.style.label.value}"
+            },
+            "color": {
+                "value": "{font.color.base.value}"
+            }
+        },
+        "disabled": {
+            "color": {
+                "value": "{font.color.inverted.value}"
+            },
+            "border-color": {
+                "value": "{border.color.base.disabled.value}"
+            },
+            "background-color": {
+                "value": "{background.color.base.disabled.value}"
+            }
+        },
+        "hover": {
+            "border-color": {
+                "value": "{border.color.base.hover.value}"
+            },
+            "background-color": {
+                "value": "{background.color.base.default.value}"
+            }
+        },
+        "active": {
+            "border-color": {
+                "value": "{border.color.base.focus.value}"
+            },
+            "background-color": {
+                "value": "{background.color.base.default.value}"
+            },
+            "box-shadow": {
+                "value": "{box-shadow.inset.focus.progressive.value}"
+            }
+        },
+        "error": {
+            "border-color": {
+                "value": "{border.color.feedback.error.value}"
+            }
+        },
+        "warning": {
+            "border-color": {
+                "value": "{color.utility.yellow.30.value}",
+                "comment": "We want to use an alias token here but it wasn't decided at the time we implemented this."
+            }
+        },
+        "desktop": {
+            "padding-inline": {
+                "value": "{dimension.spacing.medium.value}"
+            },
+            "padding-block": {
+                "value": "{dimension.padding.vertical.medium.value}",
+                "comment": "to be replaced by spacing token in the future"
+            }
+        },
+        "mobile": {
+            "padding-inline": {
+                "value": "{dimension.spacing.medium.value}"
+            },
+            "padding-block": {
+                "value": "{dimension.padding.vertical.large.value}",
+                "comment": "to be replaced by spacing token in the future"
+            }
+        },
+        "transition": {
+            "duration": {
+                "value": "{transition.interaction.field.duration.value}"
+            },
+            "property": {
+                "value": "{transition.interaction.field.property.value}"
+            },
+            "timing-function": {
+                "value": "{transition.timing-function.ease.value}"
+            }
+        },
+        "menu": {
```

Comment:
Since we are not using a native select after all, but are reusing the LookupMenu, these `menu` tokens are probably no longer needed. Sorry for the extra work :pensive:

Reply:
thanks for all the heads-up! this file really needed a review 😄